util/chunk/column.go

```diff
@@ -58,7 +58,6 @@ func (c *Column) AppendSet(set types.Set) {
 // See https://arrow.apache.org/docs/memory_layout.html
 type Column struct {
 	length     int
-	nullCount  int
```

Comment:
why do we remove this?

Reply:
The description is updated, and here is the reason:

Remove `nullCnt` field in Column since it's hard to maintain after we have methods like `Int64s() []int64` which allow users to modify values of the column outside.
On the other hand, `nullCnt` is only used when we encode a column, so it is not worth maintaining it.
Therefore, I replaced it with a method `NullCount()`.

Reply:
I do agree to remove `nullCount`. Irrelevant about this PR, I think we should use `Int64s` together with null bitmap.

Reply:
You mean that change `Int64s() []int64` to `Int64s() (i64s []int64, bitmap []byte)` and let the user update the `bitmap` outside?

Reply:
No, I mean when they perform some expressions, such as ifnull(a, b), if `a[i]` is null, result should be `b[i]`. Here the expression evaluation should know whether the i-th element is null.

Reply:
There is another method `IsNull`:
```
for _, idx := range sel {
    v := a[idx]
    if colA.IsNull(idx) {
        v = b[idx]
    }
    ...
}
```